settings.gradle.kts

```diff
@@ -5,12 +5,10 @@
 rootProject.name = "kotlinx.reflect.lite"
 
 pluginManagement {
-    resolutionStrategy {
-        val kotlin_version: String by settings
-        eachPlugin {
-            if (requested.id.namespace?.startsWith("org.jetbrains.kotlin") == true) {
-                useVersion(kotlin_version)
```

Comment:
Was causing any plugin beginning with `org.jetbrains.kotlin` (including `kotlinx` and thus binary compatibility validator) to be using `kotlin_version`